infection.json

```diff
@@ -7,15 +7,33 @@
             "Kernel.php"
         ]
     },
-    "timeout": 10,
+    "timeout": 1000,
```

Comment:
This won't increase too much the checking time for infection?

Reply:
Actually this is a bug I think, in the documentation it says seconds but I'm 100% sure it's not in seconds. I was playing with the numbers and this was the magic number where I didn't get timeouts, and as you can see in the CI report it has no impact on the overall time (which is weird I admit)